plugins/chat/lib/direct_message_channel_creator.rb

```diff
@@ -11,6 +11,7 @@ def self.create!(acting_user:, target_users:)
     if direct_message
       chat_channel = ChatChannel.find_by!(chatable: direct_message)
     else
+      enforce_max_direct_message_users!(acting_user, target_users)
```

Comment:
Not related to your change, but I find this `if` a bit confusing. The else here means this is a regular message, not a DM. 😕

Reply:
It just means it already is existing for these users no?

Reply:
You're right. My bad, sorry.

Reply:
no worries, thanks

Reply:
I see what @romanrizzi means...maybe we should rename it `direct_message_chatable`? Then we can do `if direct_message_chatable.present?` and it's a bit clearer. Anyway not a huge deal, just saying I had the same thought here